src/Services/Manager.vala

```diff
@@ -170,6 +175,16 @@ public class Bluetooth.Services.ObjectManager : Object {
         }
     }
 
+    public string? get_name () {
```

Comment:
```suggestion
    public unowned string? get_name () {
```

Reply:
I get a compile error if I do this:
```
Manager.vala:183.17-183.64: error: Return value transfers ownership but method return type hasn't been declared to transfer ownership

```

Reply:
Ah yeah haven't saw that it was owned here, so everything is fine